commands/dashboard/index.js

```diff
@@ -0,0 +1,131 @@
+    return apps.map(app => {
+      let types = app.formation.map(p => p.type);
+      return {
+        dynoErrors:    types.map(type => heroku.request({host: 'api.metrics.herokai.com', path: `/apps/${app.app.name}/formation/${type}/metrics/errors?${date}`, headers: {Range: ''}}).catch(() => null)),
+        routerLatency: heroku.request({host: 'api.metrics.herokai.com', path: `/apps/${app.app.name}/router-metrics/latency?${date}&process_type=${types[0]}`, headers: {Range: ''}}).catch(() => null),
+        routerErrors:  heroku.request({host: 'api.metrics.herokai.com', path: `/apps/${app.app.name}/router-metrics/errors?${date}&process_type=${types[0]}`, headers: {Range: ''}}).catch(() => null),
+        routerStatus:  heroku.request({host: 'api.metrics.herokai.com', path: `/apps/${app.app.name}/router-metrics/status?${date}&process_type=${types[0]}`, headers: {Range: ''}}).catch(() => null),
+      };
+    });
+  }
+
+  function* displayApps (apps) {
+    let owner = owner => owner.email.endsWith('@herokumanager.com') ? owner.email.split('@')[0] : owner.email;
+
+    let metrics = yield fetchMetrics(apps);
+    for (let a of _.zip(apps, metrics)) {
+      cli.log();
+      let app     = a[0];
+      let metrics = a[1];
+      cli.log(cli.color.app(app.app.name));
+      let pipeline = app.pipeline ? ` Pipeline: ${cli.color.blue(app.pipeline.pipeline.name)}`: ``;
+      cli.log(`  Owner: ${cli.color.blue(owner(app.app.owner))}${pipeline}`);
+      displayFormation(app.formation);
+      cli.log(`  Last release: ${cli.color.green(time.ago(new Date(app.app.released_at)))}`);
+      displayMetrics(metrics);
+      displayErrors(metrics);
+    }
+  }
+
+  function displayNotifications (notifications) {
+    notifications = notifications.filter(n => !n.read);
+    if (notifications.length > 0) {
+      cli.log(cli.color.yellow(`\nYou have ${notifications.length} unread notifications. Read them with ${cli.color.blue('heroku notifications')}`));
+    }
+  }
+
+  let apps = yield favoriteApps();
+
+  let data = yield {
+    orgs:          heroku.request({path: '/organizations', partial: true, headers: {Range: 'id ..; max=1'}}),
+    notifications: heroku.request({host: 'telex.heroku.com', path: '/user/notifications'}),
+    apps:    apps.map(app => ({
+      app:         heroku.get(`/apps/${app}`),
+      formation:   heroku.get(`/apps/${app}/formation`),
+      pipeline:    heroku.get(`/apps/${app}/pipeline-couplings`).catch(() => null),
+    })),
+  };
+
+  if (apps.length > 0) yield co(displayApps(data.apps));
+  else cli.warn(`Add apps to this dashboard by favoriting them with ${cli.color.blue('heroku apps:favorites:add')}`);
+
+  cli.log(`\nSee all add-ons with ${cli.color.blue('heroku addons')}`);
+  let sampleOrg = _.sortBy(data.orgs, o => new Date(o.created_at))[0];
+  if (sampleOrg) cli.log(`See all apps in ${cli.color.blue(sampleOrg.name)} with ${cli.color.blue('heroku apps --org ' + sampleOrg.name)}`);
```

Comment:
@dickeyxxx I was testing this and it listed an org that I am a collaborator on, but do not have access to the app listing to.  We should narrow this down to just those that would actually allow app listings.  Holler at me in chat and I can find the code in the backend that matches.